samples/config_helper_vulkan.cc

```diff
@@ -1242,52 +1249,102 @@ amber::Result ConfigHelperVulkan::CreateDeviceWithFeatures2(
+        features_.variable_pointers.variablePointers = VK_TRUE;
+      } else if (feature == kVariablePointersStorageBuffer) {
+        features_.variable_pointers.variablePointersStorageBuffer = VK_TRUE;
+      }
+    } else if (StartsWith(feature, "Float16Int8Features.")) {
+      init_feature(supports_.shader_float16_int8, features_.float16_int8,
+                   VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FLOAT16_INT8_FEATURES_KHR,
+                   VK_KHR_SHADER_FLOAT16_INT8_EXTENSION_NAME);
+      if (feature == kFloat16Int8_Float16) {
+        features_.float16_int8.shaderFloat16 = VK_TRUE;
+      } else if (feature == kFloat16Int8_Int8) {
+        features_.float16_int8.shaderInt8 = VK_TRUE;
+      }
+    } else if (StartsWith(feature, "Storage8BitFeatures.")) {
+      init_feature(supports_.shader_8bit_storage, features_.storage_8bit,
+                   VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_8BIT_STORAGE_FEATURES_KHR,
+                   VK_KHR_8BIT_STORAGE_EXTENSION_NAME);
+      if (feature == k8BitStorage_Storage) {
+        features_.storage_8bit.storageBuffer8BitAccess = VK_TRUE;
+      } else if (feature == k8BitStorage_UniformAndStorage) {
+        features_.storage_8bit.uniformAndStorageBuffer8BitAccess = VK_TRUE;
+      } else if (feature == k8BitStorage_PushConstant) {
+        features_.storage_8bit.storagePushConstant8 = VK_TRUE;
+      }
+    } else if (StartsWith(feature, "Storage16BitFeatures.")) {
+      init_feature(supports_.shader_16bit_storage, features_.storage_16bit,
+                   VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_16BIT_STORAGE_FEATURES_KHR,
+                   VK_KHR_16BIT_STORAGE_EXTENSION_NAME);
+      if (feature == k16BitStorage_Storage) {
+        features_.storage_16bit.storageBuffer16BitAccess = VK_TRUE;
+      } else if (feature == k16BitStorage_UniformAndStorage) {
+        features_.storage_16bit.uniformAndStorageBuffer16BitAccess = VK_TRUE;
+      } else if (feature == k16BitStorage_PushConstant) {
+        features_.storage_16bit.storagePushConstant16 = VK_TRUE;
+      } else if (feature == k16BitStorage_InputOutput) {
+        features_.storage_16bit.storageInputOutput16 = VK_TRUE;
+      }
+    } else if (StartsWith(feature, "SubgroupSizeControl.")) {
+      init_feature(
+          supports_.subgroup_size_control, features_.subgroup_size_control,
+          VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SUBGROUP_SIZE_CONTROL_FEATURES_EXT,
+          VK_EXT_SUBGROUP_SIZE_CONTROL_EXTENSION_NAME);
+      if (feature == kSubgroupSizeControl) {
+        features_.subgroup_size_control.subgroupSizeControl = VK_TRUE;
+      } else if (feature == kComputeFullSubgroups) {
+        features_.subgroup_size_control.computeFullSubgroups = VK_TRUE;
+      }
+    } else if (feature == kShaderSubgroupExtendedTypes) {
+      init_feature(
+          supports_.shader_subgroup_extended_types,
+          features_.shader_subgroup_extended_types,
+          VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_SUBGROUP_EXTENDED_TYPES_FEATURES,
```

Comment:
Clang-format is fine with this, but the linter complains that it is more than 80 characters wide. It is hard to keep it to 80 when the macro is 73 characters. There are something we can do. Let me now if you have strong thoughts on how to handle it.

Reply:
I would recommend turning the linter off for these lines.

Reply:
Good idea. Done.